commons/src/main/java/io/aiven/kafka/connect/common/config/TransformerFragment.java

```diff
@@ -73,4 +81,22 @@ public Class<?> getAvroValueSerializer() {
         return cfg.getClass(AVRO_VALUE_SERIALIZER);
     }
 
+    public int getTransformerMaxBufferSize() {
+        return cfg.getInt(TRANSFORMER_MAX_BUFFER_SIZE);
+    }
+
+    private static class ByteArrayTransformerMaxBufferSizeValidator implements ConfigDef.Validator {
+        @Override
+        public void ensureValid(final String name, final Object value) {
+
+            // ConfigDef will throw an error if this is not an int that is supplied
+            final int size = (int) value;
+            if (size <= 0) {
```

Comment:
```suggestion
           if ((Integer) value < 0) {
```
and size variable can be deleted